index.bs

```diff
@@ -16,13 +16,448 @@ WPT Display: closed
+ability or inability to access cookies to a server. It is a [=Structured Field=] [=structured field/item=] whose value MUST be a
+[=structured field/token=]. [[!I-D.structured-field-values-for-http]] Its ABNF is:
+
+```
+Sec-Fetch-Storage-Access = sf-token
+```
+
+Valid `Sec-Fetch-Storage-Access` values include "<code>[=storage access status/none=]</code>", "<code>[=storage access status/inactive=]</code>", and
+"<code>[=storage access status/active=]</code>". In order to support forward-compatibility with as-yet-unknown
+semantics, servers SHOULD ignore this header if it contains an invalid value.
+
+<pre class="example" id="sec-fetch-storage-access-usage">
+// When the request's credentials mode is "`omit`", the header is omitted:
+<no-header>
+
+// When the request is same-site, the header is omitted:
+<no-header>
+
+// When the request has no access to unpartitioned cookies, the header's value is "<code>[=storage access status/none=]</code>":
+Sec-Fetch-Storage-Access: none
+
+// When the request has no access to unpartitioned cookies, but
+// 'storage-access' permission has already been granted, the header's value is
+// "<code>[=storage access status/inactive=]</code>":
+Sec-Fetch-Storage-Access: inactive
+
+// When the request has access to unpartitioned cookies, the header's value is "<code>[=storage access status/active=]</code>":
+Sec-Fetch-Storage-Access: active
+</pre>
+
+<div algorithm>
+    To <dfn export lt="set-storage-access">set the `Sec-Fetch-Storage-Access` header</dfn> for a [=request=] |request|:
+
+    <ol class="algorithm">
+        1.  [=Assert=]: |request|'s [=request/url=] is a [=potentially trustworthy URL=].
+        1.  If |request|'s [=request/credentials mode=] is not "`include`", abort these steps.
+        1.  Let |access| be |request|'s [=request/storage access status=].
+        1.  If |access| is null, abort these steps.
+        1.  Let |value| be a [=Structured Field=] value whose value is a [=structured field/token=].
+        1.  Set |value|'s value to |access|.
+        1.  [=header list/Set a structured field value=] given ("`Sec-Fetch-Storage-Access`", |value|) in |request|'s [=request/header list=].
+    </ol>
+</div>
+
+
+The `Activate-Storage-Access` HTTP Response Header {#activate-storage-access-header}
+-------------------------------------------------------------------------------------
+
+The <dfn http-header export>`Activate-Storage-Access`</dfn> HTTP response header
+allows a server to opt in to accessing its unpartitioned cookies in a cross-site request
+context. It is a [=Structured Field=] [=structured field/item=] whose value MUST be a [=structured
+field/token=]. [[!I-D.structured-field-values-for-http]] Its ABNF is:
+
+```
+Activate-Storage-Access = sf-item
+```
+
+Valid `Activate-Storage-Access` values include "`load`" and "`retry`". In order to support forward-compatibility with as-yet-unknown semantics, user agents SHOULD ignore this header if it contains an invalid value.
```

Comment:
```suggestion
Valid `Activate-Storage-Access` values include "`load`" and "`retry`".
```

I think this just conflicts with https://github.com/privacycg/storage-access-headers/pull/20/files#diff-5e793325cd2bfc452e268a4aa2f02b4024dd9584bd1db3c2595f61f1ecf7b985R254 where the UA is actually defined to ignore unknown values. Since this is concretely defined in the processing model, I'd remove this.

Reply:
SGTM, done